### PR TITLE
fix #61293 gorodarmavir.ru

### LINKED ISSUE #61293: gorodarmavir.ru
### Issue URL (Ads)
 [https://gorodarmavir.ru/news/kultura-i-sport/avtogonshchiki-armavira-zavoevali-dva-1-mesta-na-sorevnovaniyakh-po-dzhippingu/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fgorodarmavir.ru%2Fnews%2Fkultura-i-sport%2Favtogonshchiki-armavira-zavoevali-dva-1-mesta-na-sorevnovaniyakh-po-dzhippingu%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (2880), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Security:</b><br/> Malware Domains Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Russian
Other extensions used: | privacy badger, https everywhere, dark reader, bttv

### PR DESCRIPTION
**Pull request by stanislav-atr**
#61293

### CODE REVIEW

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -426,6 +426,7 @@ gtorrent.org##.box-style-light-banner
 gotoshop.ua##.welcome2
 knigilub.ru##.forum-body > table > tbody > tr > td[id="r-colonka"]
 gorodarmavir.ru,015.by##div[id^="banner-slider-"]
+gorodarmavir.ru##.container-for-shadow-root
```
Comment (Sergey-Lyapin) [inline RussianFilter/sections/specific.txt:429]: It's better to block adserver
`||yxswtummev.info^$third-party`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Comment by Sergey-Lyapin** [comment]
Please also block leftovers
<details>
<summary>Please also block leftovers like this</summary>

</details>

File: RussianFilter/sections/specific.txt (hunk 1 above)
Comment (Sergey-Lyapin) [reply RussianFilter/sections/specific.txt:429]: `||metrics0.com^$third-party`
tracking server

**Comment by Alex-302** [comment]
there are a lot of similar sites. Look for `/section/banners/upload/*$domain=015.by******` rule

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**